### ISSUE TITLE
train issue

### ISSUE THREAD

**Issue by ZzzjzzZ** [hellochick/PSPNet-tensorflow#13]
Hi, hellochick,
    I am very interested in your proj. And I try to train the architecture on Pascal Voc2012 dataset. But I found the training loss was around 12 during training.
    I do not load the pretrained model you provided, for the difference about the dataset. I want to know whether the weights was updated during training after I found that code which listed as follows:
   `# Gets moving_mean and moving_variance update operations from tf.GraphKeys.UPDATE_OPS
    if args.update_mean_var == False:
        update_ops = None
    else:
        update_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS)

    with tf.control_dependencies(update_ops):
        opt_conv = tf.train.MomentumOptimizer(learning_rate, args.momentum)
        opt_fc_w = tf.train.MomentumOptimizer(learning_rate * 10.0, args.momentum)
        opt_fc_b = tf.train.MomentumOptimizer(learning_rate * 20.0, args.momentum)

        grads = tf.gradients(reduced_loss, conv_trainable + fc_w_trainable + fc_b_trainable)
        grads_conv = grads[:len(conv_trainable)]
        grads_fc_w = grads[len(conv_trainable): (len(conv_trainable) + len(fc_w_trainable))]
        grads_fc_b = grads[(len(conv_trainable) + len(fc_w_trainable)):]

        train_op_conv = opt_conv.apply_gradients(zip(grads_conv, conv_trainable))
        train_op_fc_w = opt_fc_w.apply_gradients(zip(grads_fc_w, fc_w_trainable))
        train_op_fc_b = opt_fc_b.apply_gradients(zip(grads_fc_b, fc_b_trainable))

        train_op = tf.group(train_op_conv, train_op_fc_w, train_op_fc_b)`
        
        I set  args.update_mean_var to both False and True, and I found it did not work.
        How can I get over the problem. Thanks!

**Comment by hellochick** [comment]
Hey @CarryJzzZ , I think the segmentation problem is based on the classification problem. Which means that the network should learn how to classify a object well. So I think maybe you should load the weight from pre-trained ImageNet ResNet101 first. But this is only my advice, I haven't trained on the Pascal voc. Hope you will success !

**Comment by LongLong-Jing** [comment]
@CarryJzzZ, Have you successfully train the network on VOC dataset?
